doc/api/dgram.md

```diff
@@ -239,8 +239,9 @@ respect to [byte length][] and not the character position.
 If `msg` is an array, `offset` and `length` must not be specified.
 
 The `address` argument is a string. If the value of `address` is a host name,
-DNS will be used to resolve the address of the host. If the `address` is not
-specified or is an empty string, `'127.0.0.1'` or `'::1'` will be used instead.
+DNS will be used to resolve the address of the host. Otherwise, if `address` is
+not specified or is an empty string, `'127.0.0.1'` (`udp4`) or `'::1'` (`udp6`)
```

Comment:
What does "not specified" mean? I think in js we "provide" function arguments, not "specify" them. That aside, I don't understand from the text the behaviour of:

- `send("msg", 7)` // "not specified", I think
- `send("msg", 7, ()=>{}) // also not specified, I think
- `send("msg", 7, undefined) // specified as undefined, and handled by code below, I think
- `send("msg", 7, undefined, ()-.{}) // ditto
- `send("msg", 7, "") // same as if address is undefined
- `send("msg", 7, null) // not same as if address is `undefined`? This is weird and inconsistent, if true

I would state that if address is not provided, or is null, undefined, or an empty string, it will use the protocol specific localhost address - and implement and test this condition if it is not already